recipe/meta.yaml

```diff
@@ -24,18 +24,19 @@ requirements:
     - pip
     - setuptools >=40.6.0
     - wheel
-    # Required to compile .proto files during setup.py execution
-    # protoc is provided by protobuf. We do not have a package protoc-wheel.
-    - protobuf
+    # Snowpark IR
+    - protobuf ==3.20.1  # [py<=310]
+    - protobuf ==4.25.3  # [py>310]
```

Comment:
What is the rationale behind this change ?

Reply:
They did update their recipe file with those pinnings. Even though that file is not used by us or them, it probably represents their intention to pin `protobuf` to those versions for some reason. They probably could have used `setup_requires` for that.

https://github.com/snowflakedb/snowpark-python/blob/v1.26.0/recipe/meta.yaml#L29-L32

Reply:
They ditched `protoc` and therefore I also removed the comment above `protobuf`.

https://github.com/snowflakedb/snowpark-python/blob/v1.26.0/setup.py#L28

Reply:
I added a reference comment.

Reply:
I see, that makes sense. Thank you !